src/App.css

```diff
@@ -2,18 +2,37 @@
   text-align: center;
 }
 
+.app-container {
+  padding: 0 8% 0;
+}
+
+.mt-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 4em;
+  background-color: #111727;
```

Comment:
nit: for production-ready code, we usually use variables, or ideally, if combined with library components, there would be a color palette to handle most of the color variations.